src/main/resources/templates/db_scripts/removeOldPrecedingSucceedingTitles.sql

```diff
@@ -0,0 +1,9 @@
+START TRANSACTION;
+
+DELETE FROM ${myuniversity}_${mymodule}.instance_relationship
+WHERE instanceRelationshipTypeId ='cde80cc2-0c8b-4672-82d4-721e51dcb990';
```

Comment:
Are any existing records migrated to the new structure before they are deleted?

Reply:
The migration has already been done https://github.com/folio-org/mod-inventory-storage/pull/404

Reply:
Ok, thank you